core/gc/src/trace.rs

```diff
@@ -179,7 +179,7 @@ simple_empty_finalize_trace![
     char,
     TypeId,
     String,
-    Box<str>,
```

Comment:
Don't remove the implementation on `Box<str>` even if we don't use it. The users of `boa` would have to implement `Trace` for a wrapper around `Box<str>` if they had it on their structs that implement `Trace`.

Reply:
I got error from `ra` that implement `Trace` for `Cow<'static, str>` requires that `str` implements `Trace` and I tested that `Box<str>` does not affected by this.

Reply:
Oh, I think that's because we already implement `Trace` for `Box<T>`. Makes sense.